README.md

```diff
@@ -69,11 +69,9 @@ using SliceSampling
     end
 end
 
-sampler = Turing.Experimental.Gibbs(
-    (
-        p = externalsampler(SliceSteppingOut(2.0)),
-        z = PG(20, :z)
-    )
+sampler = Turing.Gibbs(
+    p => externalsampler(SliceSteppingOut(2.0)),
+    z => PG(20, :z),
```

Comment:
```suggestion
    :p => externalsampler(SliceSteppingOut(2.0)),
    :z => PG(20, :z),
```
Optionally you could use `@varname(p)` and `@varname(z)`. It's a matter of taste which you prefer.